src/libstd/sys/windows/mutex.rs

```diff
@@ -22,7 +22,7 @@
 //!    more details.
 //!
 //! 3. While CriticalSection is fair and SRWLock is not, the current Rust policy
-//!    is there there are no guarantees of fairness.
+//!    is there are no guarantees of fairness.
```

Comment:
This should actually be "the current Rust policiy is that there are no guarantees of fairness."